scripts/testdata/satellite-config.yaml.lock

```diff
@@ -241,6 +241,9 @@ compensation.withheld-percents: 75,75,75,50,50,50,25,25,25,0,0,0,0,0,0
 # indicates if new access grant flow should be used
 # console.new-access-grant-flow: false
 
+# indicates if new billing screen should be used
```

Comment:
did you add it manually? Config lock test is failing so please try to run `go generate ./satellite/...` from root folder and revert all the unnecessary changes that will appear (except changes for config-lock file)

Reply:
re-implemented using `go generate ./satellite` from root